README.md

```diff
@@ -3,6 +3,12 @@
 [![All Contributors](https://img.shields.io/badge/all_contributors-3-orange.svg?style=flat-square)](#contributors-)
 <!-- ALL-CONTRIBUTORS-BADGE:END -->
 Git & Github training exercises for when interacting with a repository that is not yours.
```

Comment:
Remove this line "Git & Github training exercises for when interacting with a repository that is not yours."

Reply:
Also you are not sending to develop. change that too.